Workshops.KernelAi.ConsoleApp/Modules/SemanticKernel/12_InlineTemplates.cs

```diff
@@ -0,0 +1,149 @@
+namespace Workshops.KernelAi.ConsoleApp.Modules.SemanticKernel;
+
+public class InlineTemplates(IAnsiConsole console, WorkshopSettings settings) : IExample
+{
+    public string Name => "Semantic Kernel Inline Templates";
+
+    public WorkshopModule Module => WorkshopModule.SemanticKernel;
+
+    public async Task RunAsync()
+    {
+        ModelSettings chatSettings = settings.Chat;
+        console.WriteModelInfo(chatSettings);
+
+        Kernel kernel = Kernel.CreateBuilder()
+            .AddWorkshopChatCompletion(chatSettings)
+            .Build();
+
+        // Template with simple variable substitution
+        string simpleTemplate = """
+            You are a creative writing assistant.
+            Write a {{$genre}} story about {{$character}} in {{$setting}}.
+            The story should be {{$length}} and have a {{$mood}} tone.
+            Keep it engaging and appropriate for all audiences.
+            """;
+
+        // Template with conditional logic
+        string conditionalTemplate = """
+            You are a helpful travel assistant.
+            {{#if hasPassport}}
+            Great! Since you have a passport, here are international travel recommendations for {{$destination}}.
+            {{else}}
+            Since you don't have a passport, here are domestic travel recommendations for {{$destination}}.
+            {{/if}}
+            
+            Budget: {{$budget}}
+            Duration: {{$duration}}
+            Interests: {{$interests}}
+            
+            Provide 3-4 specific recommendations with brief descriptions.
+            """;
+
+        // Template with loops
+        string listTemplate = """
+            You are a meal planning expert.
+            Create a weekly meal plan based on these preferences:
+            
+            Dietary restrictions: {{$dietary}}
+            Cooking skill level: {{$skillLevel}}
+            Budget per week: {{$budget}}
+            
+            Available ingredients:
+            {{#each ingredients}}
+            - {{this}}
+            {{/each}}
+            
+            Please provide a 7-day meal plan with breakfast, lunch, and dinner.
+            Include simple recipes for dishes that use the available ingredients.
+            """;
+
+        // Create kernel functions from templates
+        KernelFunction storyGenerator = kernel.CreateFunctionFromPrompt(simpleTemplate, functionName: "StoryGenerator");
+        KernelFunction travelPlanner = kernel.CreateFunctionFromPrompt(conditionalTemplate, functionName: "TravelPlanner");
```

Comment:
This generates a KernelException: A function named argument must contain a name and value separated by a '=' character.